src/Extension.php

```diff
@@ -859,6 +863,13 @@ public function update_payment( Payment $payment ) {
 			return;
 		}
 
+		$this_pronamic_payment_id = (string) $payment->get_id();
+		$last_pronamic_payment_id = (string) \rcp_get_payment_meta( $payment->get_source_id(), '_pronamic_payment_id', true );
+
+		if ( ! empty( $last_pronamic_payment_id ) && $this_pronamic_payment_id !== $last_pronamic_payment_id ) {
```

Comment:
@rvdsteege Don't us `empty`?
https://dev.to/aleksikauppila/using-isset-and-empty-hurts-your-code-aaa
Doesn't PHPStan also advise not to use this?